domain/user.js

```diff
@@ -8,8 +8,8 @@ class User {
     throw new Error(`${this.viewProfile.name} is not implemented.`);
   }
 
-  updateProfile() {
-    throw new Error(`${this.updateProfile.name} is not implemented.`);
+  async updateProfile(mongo, obj) {
```

Comment:
I would use a more descriptive name for obj.

Reply:
ok, let me update